code/modules/events/holiday/xmas.dm

```diff
@@ -99,8 +99,8 @@
 			var/response = alert(M, "Santa is coming to town! Do you want to be santa?", "Ho ho ho!", "Yes", "No")
 			if(response == "Yes" && M && M.client && M.stat == DEAD && !santa)
 				santa = new /mob/living/carbon/human(pick(blobstart))
-				M.mind.transfer_to(santa)
 				santa.key = M.key
+				qdel(M)

```

Comment:
qdel(mob) calls del(mob) anyway, but also calls ghostize() and some other cleanup procs.